src/sortStudents.ts

```diff
@@ -1,16 +1,56 @@
 
 export interface Student {
-  // describe Student interface
+  name: string,
+  surname: string,
+  age: number,
+  married: boolean,
+  grades: number[],
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name = 'name',
+  Surname = 'surname',
+  Age = 'age',
+  Married = 'married',
+  AverageGrade = 'grades',
 }
 
-// create SortOrder type
-export type SortOrder;
+export type SortOrder = 'asc' | 'desc';
 
+function getAverage(students: Student): number {
+  return students.grades.reduce((a, b) => a + b) / students.grades.length;
```

Comment:
Рекомендую завжди задавати початкове значення для `reduce`, це робить код більш передбачуваним))
```suggestion
  return students.grades.reduce((a, b) => a + b, 0) / students.grades.length;
```